defs/scenarios/mysql/bugs.yaml

```diff
@@ -0,0 +1,34 @@
+checks:
+  has_1971565:
+    requires:
+      apt:
+        mysql-router:
+          # focal:
+          - min: 8.0.29-0ubuntu0.20.04.2
+            max: 8.0.29-0ubuntu0.20.04.2
+          # impish:
+          - min: 8.0.29-0ubuntu0.21.10.1
+            max: 8.0.29-0ubuntu0.21.10.1
+          # jammy:
+          - min: 8.0.29-0ubuntu0.22.04.1
+            max: 8.0.29-0ubuntu0.22.04.1
+  mysql_router_enabled:
+    requires:
+      systemd: mysql-router
```

Comment:
So, this will test that mysql-router service exists but will not check state. Do you think we should check that it is actually enabled or not?

e.g.

mysql_router_enabled:
  requires:
    systemd:
      mysql-router: enabled